### ISSUE TITLE
Unknown architechture + multiple English languages

### ISSUE THREAD

**Issue by DotTheI** [IngmarStein/Monolingual#44]
Version 1.70 doesn't recognize my computer's architecture
![screen shot 2015-09-21 at 12 58 10 pm](https://cloud.githubusercontent.com/assets/10737439/10003315/a3b949c0-6060-11e5-80b2-f710d18a6ef3.png)

Also, there are now dozens of English languages for almost every country. What are those?
![screen shot 2015-09-21 at 1 01 04 pm](https://cloud.githubusercontent.com/assets/10737439/10003363/d6917ffc-6060-11e5-9c4d-84265ff42448.png)

**Comment by swiftyvee** [comment]
This screwed up my os x install completely. It normally detects what architecture i have installed and without looking i ran it and it removed all my 64-bit files to everything. it screwed everything up! I am so so shattered right now. This really needs to be fixed or at least warn the people.

**Comment by IngmarStein** [comment]
@DotTheI so, what is your computer's architecture?

**Comment by DotTheI** [comment]
@IngmarStein It is Intel 64-bit

**Comment by IngmarStein** [comment]
Could you please tell me the following sysctl values on your system?

    $ sysctl hw.cputype
    hw.cputype: 7
    $ sysctl hw.cpusubtype
    hw.cpusubtype: 8

**Comment by turtlix** [comment]
"unknown" architecture. Slicing my OSX installation :(
```
$ sysctl hw.cputype
hw.cputype: 7
$ sysctl hw.cpusubtype
hw.cpusubtype: 4
```

**Comment by swiftyvee** [comment]
I had to restore all my apps and os x installation from a backup to get up and running again. =(

**Comment by DotTheI** [comment]
@IngmarStein How do I get those?

**Comment by IngmarStein** [comment]
Will be fixed in 1.7.1

Btw, the language list now comes from OS X and it now that many English variations.
